### PR TITLE
intensity-normalization: init at 2.0.1

### PR DESCRIPTION
**Pull request by bcdarwin**
python3Packages.scikit-fuzzy:  unstable-2020-10-03 -> unstable-2021-03-31

###### Motivation for this change

Add a package.

Since this package was primarily intended as a CLI package but now has a Python API, I've added to both `python-packages` and `all-packages`.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [NA] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`): some binaries fail with an ImportError due to missing optional `antspy` dependency
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [NA] (Package updates) Added a release notes entry if the change is major or breaking
  - [NA] (Module updates) Added a release notes entry if the change is significant
  - [NA] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
`nixpkgs-review` happy, macos 10.15 & nixos x86_64

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136487` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>intensity-normalization (python39Packages.intensity-normalization)</li>
    <li>python38Packages.intensity-normalization</li>
    <li>python38Packages.scikit-fuzzy</li>
    <li>python39Packages.scikit-fuzzy</li>
  </ul>
</details>

File: pkgs/development/python-modules/intensity-normalization/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/jcreinhold/intensity-normalization";
+    description = "MRI intensity normalization tools";
+    maintainers = with maintainers; [ bcdarwin ];
+    license = licenses.asl20;
+  };
+
+}
```
Comment (fabaff) [inline pkgs/development/python-modules/intensity-normalization/default.nix:49]:
```suggestion
  };
}
```

File: pkgs/development/python-modules/intensity-normalization/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    matplotlib
+    nibabel
+    numpy
+    scikit-fuzzy
+    scikitimage
+    scikit-learn
+    scipy
+    statsmodels
+  ];
+
```
Comment (fabaff) [inline pkgs/development/python-modules/intensity-normalization/default.nix:41]: Please add `pythonImportsCheck` as additional safe-guard.

File: pkgs/development/python-modules/intensity-normalization/default.nix (hunk 1 above)
Comment (bcdarwin) [reply pkgs/development/python-modules/intensity-normalization/default.nix]: `nixpkgs-fmt` was fine with the original, but I've changed it.

File: pkgs/development/python-modules/intensity-normalization/default.nix (hunk 2 above)
Comment (bcdarwin) [reply pkgs/development/python-modules/intensity-normalization/default.nix]: I've added this -- I wasn't aware this was now the standard for packages which have `doCheck = true` (I am aware of #133032 but once this is resolved it seems a bit unnecessary?).

Comment (fabaff) [reply pkgs/development/python-modules/intensity-normalization/default.nix:47]: It's not a "standard". I would say, it's more a "common practice" :wink:

**Review by fabaff — APPROVED** [review]
Looks good to me.

**Pull request merged**